app/scripts/modules/openstack/src/region/regionSelectField.directive.js

```diff
@@ -12,49 +12,47 @@ module.exports = angular.module('spinnaker.openstack.region.regionSelectField.di
   .directive('osRegionSelectField', function (accountService) {
     return {
       restrict: 'E',
-      templateUrl: require('./regionSelectField.directive.html'),
+      templateUrl: require('../common/cacheBackedSelectField.template.html'),
       scope: {
-        label: '@?',
-        labelColumnSize: '@?',
-        helpKey: '@?',
-        model: '<?',
+        label: '@',
+        labelColumnSize: '@',
+        helpKey: '@',
+        model: '=',
+        filter: '=',
         account: '<',
         onChange: '&',
-        readOnly: '<',
-        allowNoSelection: '<',
-        noOptionsMessage: '@?',
-        noSelectionMessage: '@?'
+        readOnly: '=',
```

Comment:
Surprised by the swap from `'<'` to `'='` here and below, since you shouldn't need `'='` unless the value of the binding is going to be changed within the directive.